packages/transport/src/modules/messages/MessageController.ts

```diff
@@ -507,10 +507,10 @@ export class MessageController extends TransportController {
                 );
             }
         } else {
-            relationship = await this.relationships.getActiveRelationshipToIdentity(envelope.createdBy);
+            relationship = await this.relationships.getOnceActiveRelationshipToIdentity(envelope.createdBy);
```

Comment:
```suggestion
            relationship = await this.relationships.getRelationshipToIdentity(envelope.createdBy);
```
this would be the way easier version without needing the new strange wording